### ISSUE TITLE
Read-model SQLite heisenbug

### ISSUE THREAD

**Issue by IhostVlad** [reimagined/resolve#1166]
_**Bug has been written from Oliver's words**_

**Describe the bug**
I just had a strange problem when I restarted my application. An error came up during startup:
```
resolve:resolve-runtime:local-event-broker Error while applying events to read-model { [Error: SQLITE_BUSY: database is locked] errno: 5, code: 'SQLITE_BUSY' } +10ms
```

This was followed later by a related issue:
```
resolve:resolve-runtime:local-event-broker Error while applying events to read-model { listenerId: 'Members',
  lastError:
   { Error: SQLITE_ERROR: no such table: defaultMembers
     message: 'SQLITE_ERROR: no such table: defaultMembers',
     stack: 'Error: SQLITE_ERROR: no such table: defaultMembers' },
  lastEvent: null } +4ms
```

I started the app twice with the same problem (removing data/* in between runs). I looked around and double-checked my configuration comparing to a standard new application and the hacker-news sample. I couldn't find any relevant differences, so I ran the app again - and suddenly the errors were gone!

**To Reproduce**
I have tried this a few more times and I think it might be connected. Very strange behavior, but..... so basically I do this:
- run sqlite3 data/read-models.db
- do something - "select * from ..."
- quit sqlite tool (Ctrl-D)
- rm -f data/*
- ll data - this shows that there's only .gitignore left in there
- Now run the resolve app 

I can't be completely sure, but I have seen a few times now that errors came up under these circumstances. So.... that's weird?!? Does sqlite somehow retain a lock on the db file that doesn't even exist anymore?

**Desktop (please complete the following information):**
 - OS: Linux
 - Version 0.20.21

**Comment by IhostVlad** [comment]
```
C:\Users\vladv\resolve\examples\hacker-news>C:\Users\vladv\Downloads\sqlite-tools-win32-x86-3280000\sqlite-tools-win32-x86-3280000\sqlite3.exe data\read-models.db
SQLite version 3.28.0 2019-04-16 19:49:53
Enter ".help" for usage hints.
sqlite> select * from defaultStories;
1|"7897987"|"8789789789"|""|0|[]|1564992810810|"34970414-9d79-4233-9eac-09fcb286d0d0"|"999999"|"26d6c0ef-8f11-4d55-a1d9-2545347dfcc7"|"ask"
sqlite>

C:\Users\vladv\resolve\examples\hacker-news>del data\*.db*

C:\Users\vladv\resolve\examples\hacker-news>yarn dev
yarn run v1.12.3
$ babel-node ./run.js dev
[ Client ] Time: 5679ms
Built at: 2019-08-05 11:14:34
[ Server local entry point ] Time: 4019ms
Built at: 2019-08-05 11:14:32
[ Local event broker entry point ] Time: 378ms
Built at: 2019-08-05 11:14:29
Application listening on port 3000!
[ Client ] Time: 4971ms
Built at: 2019-08-05 11:14:49
[ Server local entry point ] Time: 3295ms
Built at: 2019-08-05 11:14:48
[ Local event broker entry point ] Time: 329ms
Built at: 2019-08-05 11:14:45
Application listening on port 3000!

You can now view hacker-news in the browser.

  Local:            http://localhost:3000/
  On Your Network:  http://172.22.5.95:3000/

Note that the development build is not optimized.
To create a production build, use:
  yarn build.
  yarn start.
```

**Comment by IhostVlad** [comment]
Most likely fixed in version 0.20.22
Will be reopened if heisenbug will appear in future
